sa/database_test.go

```diff
@@ -58,3 +60,18 @@ func TestNewDbMap(t *testing.T) {
 	}
 
 }
+
+func TestStrictness(t *testing.T) {
+	const mysqlConnectURL = "mysql+tcp://policy:password@boulder-mysql:3306/boulder_policy_integration?readTimeout=800ms&writeTimeout=800ms"
```

Comment:
Good catch. Fixed.

On 06/06/2016 12:39 PM, Daniel McCarney wrote:
>
> In sa/database_test.go
> <https://github.com/letsencrypt/boulder/pull/1890#discussion_r65954479>:
>
> > @@ -58,3 +60,18 @@ func TestNewDbMap(t *testing.T) {
> >  	}
> >  
> >  }
> > +
> > +func TestStrictness(t *testing.T) {
> > +	const mysqlConnectURL = "mysql+tcp://policy:password@boulder-mysql:3306/boulder_policy_integration?readTimeout=800ms&writeTimeout=800ms"
>
> Is this const being referenced anywhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/letsencrypt/boulder/pull/1890/files/29d46ad8442e36b93679cfb27aa0d4ebdcf01c16#r65954479>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANcLRwJeIh8MGdWKbmAU4SmpI0D0Qlqks5qJHeNgaJpZM4IuDmK>.
>